apps/ui/src/components/Site/Footer.vue

```diff
@@ -34,10 +34,17 @@ const SOCIALS = [
               <AppLink :to="{ name: 'my-explore' }"> Explore spaces </AppLink>
             </div>
             <div>
-              <a href="https://v1.snapshot.box/#/setup" target="_blank">
+              <AppLink
+                :to="{
+                  name: 'create',
```

Comment:
This route is not valid anymore, since https://github.com/snapshot-labs/sx-monorepo/pull/1167

Reply:
What should it be? I can see that `snapshotx` was moved to `snapshot-x` but `snapshot` was not updated.

Reply:
https://github.com/snapshot-labs/sx-monorepo/pull/1167/files#diff-40f525d703de01090e2081ca638f5e2032bab5767647ad8758d4099b246f9855R53

new route name is `create-space-snapshot`, without params